package.json

```diff
@@ -94,14 +94,14 @@
 	},
 	"dependencies": {
 		"adm-zip": "^0.5.16",
-		"convert": "^5.1.0",
+		"convert": "^5.5.1",
 		"decompress": "^4.2.1",
 		"excel-to-json": "^1.0.0",
 		"fs-extra": "^11.2.0",
 		"node": "^18.17.0",
 		"node-stream-zip": "^1.15.0",
-		"unzipper": "^0.10.14",
+		"unzipper": "^0.12.3",
 		"uuid": "^7.0.3",
-		"xlsx": "^0.18.5"
+		"xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz"

```

Comment:
0.18.5 was the last version published in the typical npm way. Any higher versions are available like this:
https://docs.sheetjs.com/docs/getting-started/installation/frameworks